starlette_web/common/app.py

```diff
@@ -41,37 +38,29 @@ def __init__(self, *args, **kwargs):
 class BaseStarletteApplication:
     app_class: AppClass = WebApp
 
-    def pre_app_init(self):
+    def pre_app_init(self) -> None:
         """
         Extra actions before app's initialization (can be overridden)
         """
         pass
 
-    def post_app_init(self, app: AppClass):
+    def post_app_init(self, app: AppClass) -> None:
         """
         Extra actions after app's initialization (can be overridden)
         """
-        # TODO: remove logging to Sentry
-        if settings.SENTRY_DSN:
-            logging_integration = LoggingIntegration(level=logging.INFO, event_level=logging.ERROR)
-            sentry_sdk.init(settings.SENTRY_DSN, integrations=[logging_integration])
+        pass
 
-    @property
-    def debug(self):
+    def get_debug(self) -> bool:
         return settings.APP_DEBUG
 
-    @property
-    def middleware(self):
-        # TODO: remove sentry middleware
-        return [Middleware(SentryAsgiMiddleware)]
+    def get_middleware(self) -> List[Middleware]:
+        return []
 
-    @property
-    def routes(self) -> List[Union[WebSocketRoute, Route, Mount]]:
+    def get_routes(self) -> List[Union[WebSocketRoute, Route, Mount]]:
```

Comment:
```suggestion
    def get_routes(self) -> list[WebSocketRoute | Route | Mount]:
```
давай начнем юзать фичи 3.10 ?